composer/algorithms/cutmix/cutmix.py

```diff
@@ -281,8 +339,8 @@ def _rand_bbox(W: int,
 
     bbx1 = np.clip(cx - cut_w // 2, 0, W)
     bby1 = np.clip(cy - cut_h // 2, 0, H)
-    bbx2 = np.clip(cx + cut_w // 2, 0, W)
-    bby2 = np.clip(cy + cut_h // 2, 0, H)
+    bbx2 = np.clip(cx + (cut_w + 1) // 2, 0, W)
```

Comment:
This is a change from the way the [original implementation ](https://github.com/clovaai/CutMix-PyTorch/blob/2d8eb68faff7fe4962776ad51d175c3b01a25734/train.py#L292) does things, and it could potentially change how cutmix behaves slightly, I think? Differences would likely be minor though.

Reply:
good point; I added this so that odd `length` values would be respected, but if we remove integer length arguments, all the changes to this function can get reverted I think